components/fms-consumer/src/main.rs

```diff
@@ -251,13 +352,21 @@ pub async fn main() {
         .unwrap_or(option_env!("VERGEN_GIT_SHA").unwrap_or("unknown"));
 
     let mut parser = Command::new("FMS data consumer")
+        .arg_required_else_help(true)
         .version(version)
-        .about("Receives FMS related VSS data points via Hono's Kafka based Telemetry API and writes them to an InfluxDB server")
-        .arg(
+        .about("Receives FMS related VSS data points via Hono's Kafka based Telemetry API or Eclipse Zenoh instance and writes them to an InfluxDB server");
+
+    parser = influx_client::connection::add_command_line_args(parser);
+
+    parser = parser
+        .subcommand_required(true)
+        .subcommand(
+            Command::new(SUBCOMMAND_HONO)
+                .about("Forwards VSS data to an Influx DB server from hono").arg(
             Arg::new(PARAM_KAFKA_PROPERTIES_FILE)
                 .value_parser(clap::builder::NonEmptyStringValueParser::new())
                 .long(PARAM_KAFKA_PROPERTIES_FILE)
-                .help("The path to a file containing Kafka client properties for connecting to the Kafka broker(s).")
+                .help("The path to a file containing Kafka client properties for connecting to the Kafka broker(s).")                
```

Comment:
can you please remove the trailing whitespace?

Reply:
trailing whitespace removed